### DISCUSSION TITLE
the auth.users table rls message is confusing

### DISCUSSION THREAD

**Discussion by devhandler** [supabase/supabase#14972] [Feature Requests]
it still says enable rls is recommended, but after some google search, seems we don't need to enable it for tables in auth schema

![image](https://github.com/supabase/supabase/assets/133625693/f5e0b66f-b038-40d9-9b51-bef2446ec24c)

**Comment by devhandler** [comment]
it's also inconsistent that you can enable rls in table editor UI, but in authentication->policies, it shows it's locked (vs. public tables, you can enable/disable rls in policies)

![image](https://github.com/supabase/supabase/assets/133625693/6cb9ea52-854f-4d6b-aa7e-7c4a2b4c1077)
![image](https://github.com/supabase/supabase/assets/133625693/8edf39bf-83c0-4617-b454-4940d5789d8f)

**Reply by GaryAustin1** [reply]
Just confirming RLS is not at all useful for Auth schema as it can’t be accessed thru the API.

**Reply by devhandler** [reply]
agreed. it would be great in the UI the message could just say so instead of showing "enable RLS recommended"
